graphql/resolver.go

```diff
@@ -104,17 +106,21 @@ func New(apiURL string) Config {
 			}
 		}
```

Comment:
(request) can we move the block for `CreateProjectMutation` above the declaration of the `args` variable, since it doesn't use the `args` variable?